concepticondata/concepticon.tsv

```diff
@@ -3148,3 +3148,33 @@ ID	GLOSS	SEMANTICFIELD	DEFINITION	ONTOLOGICAL_CATEGORY	REPLACEMENT_ID
+3160	SLAP	Basic actions and technology	Hit with the open palm of the hand	Action/Process	
+3161	CHEW BETEL	Food and drink	To consume a mixture of areca nut and betel leaf.	Action/Process	
+3162	CROSS-COUSIN	Kinship	The child of a parent’s opposite-sex sibling, for example the father’s sister’s son.	Person/Thing	
+3163	SQUAT	Motion	Have bent knees with the body weight is on the feet.	Action/Process	
+3164	ONE HUNDRED AND TWENTY THREE	Quantity	The cardinal number represented in Roman numerals as CXXIII, and in Arabic numerals as 123.	Other	
+3165	TWENTY THREE	Quantity	The cardinal number represented in Roman numerals as XXIII, and in Arabic numerals as 23.	Other	
+3166	TWENTY TWO	Quantity	The cardinal number represented in Roman numerals as XXII, and in Arabic numerals as 22.	Other	
+3167	SERMON	Religion and belief	A religious lecture or speech, addressing issues of religion or morality.	Person/Thing	
+3168	DOWRY	Social and political relations	Property, gifts or money transfered from the parents of a daughter to her new family at marriage.	Person/Thing	
+3169	TREATY	Social and political relations	Formal agreement between nations, states or other international bodies.	Person/Thing	
+3170	BATHE SOMEONE	The body	Clean someone else with water.	Action/Process	
+3171	RINGWORM	The body	Dermatophytosis, fungal skin infection making round patches on the skin.	Person/Thing	
+3172	SCABIES	The body	A contagious skin rash caused by the mite Sarcoptes scabiei.	Person/Thing	
+3173	TINEA	The body	Dermatophytosis, fungal skin infection making white spots/non-symmetrical white patches on the skin.	Person/Thing	
+3174	WAKE SOMEONE UP	The body	Rouse from sleep.	Action/Process	
+3175	MOUND OF STONES FOR RITUAL ACTIVITIES	The house	A mound of stones or altar around/on which ritual activities are held.	Person/Thing	
+3176	RACK (FOR STORING FOOD)	The house	Bamboo shelves in the kitchen to store food and utensils.	Person/Thing	
+3177	RAISED PLATFORM	The house	An elevated wooden or bamboo platform on which people can sit, squat or sleep.	Person/Thing	
+3178	RITUAL GROUND	The house	An uncovered open area used for ritual activities and public meetings.	Person/Thing	
+3179	CORAL ROCK	The physical world	Limestone created from the skeleton of reef-forming stony corals (Scleractinia), displaying their growth patterns.	Person/Thing	
+3180	SHOOT WITH SLINGSHOT	Warfare and hunting	Use a rubber-based projectile weapon to propel projectiles against a target.	Action/Process	
+3181	PLANT RICE	Agriculture and vegetation	To put a rice seedling in the wet ground of a paddy so that it strikes root and grows.	Action/Process	
+3182	PLANT YAM	Agriculture and vegetation	To put a yam seed tuber into ridged-up ground so that it strikes root and grows.	Action/Process	
+3183	SARONG OF WOMAN	Clothing and grooming	The female style of a large tube or length of fabric, often wrapped around the waist, worn throughout much of South Asia. 	Person/Thing	
```

Comment:
we had some woman clothing sounding similar, maybe check our reflexes. If not write SARONG (OF WOMAN)

Reply:
How did I mess up the brackets *nearly every single time?*

Obviously, you do have SARONG, but as far as I'm told, there are languages that have different terms for SARONG (OF MAN) and SARONG (OF WOMAN). Given that we have only one of these, I'd be fine with linking this to SARONG and making the disambiguation part of the notes.

Reply:
Okay, I think, as it would otherwise require to set up SARONG -> SARONG (OF WOMAN), etc., which is then also a possibility, even if SARONG (OF MAN) is not (yet) there